scripts/src/mock_client.ts

```diff
@@ -1790,6 +1790,22 @@ async function checkClientMigration() {
 	await client.changeAppBlockchainVersion("2");
 }
 
+async function checkTransferKin(){
+	console.log("===================================== checkTransferKin =====================================");
+
+	const userId = generateId();
+	const deviceId = generateId();
+	const appClient = new SampleAppClient(SMPL_APP_CONFIG.jwtAddress);
+	let jwt = await appClient.getRegisterJWT(userId, deviceId);
+	const client = await MarketplaceClient.create({ jwt });
+	await client.updateWallet(SMPL_APP_CONFIG.keypair.publicKey());
+
+	const receiverWalletAddress = `wallet-${ generateId() }`;
+	const order = await client.createCrossAppOrder(receiverWalletAddress,'sender-app','mock client title','mock client description',1000);
```

Comment:
does this pass the linter? you're missing space after comma

Reply:
Do I need to manually config the linter? assumed it is part of the make.

Reply:
linted